UIKitDZ/UIKitDZ/BirthdayInformation.swift

```diff
@@ -0,0 +1,243 @@
+        myName = UILabel(frame: CGRect(x: 105, y: 499, width: 209, height: 20))
+        myName.text = ""
+        myName.shadowColor = .black
+        myName.shadowOffset = CGSize(width: 1, height: 1)
+        if let customFont = UIFont(name: fontName, size: 16) {
+            myName.font = customFont
+            view.addSubview(myName)
+
+            myFotosImageView = UIImageView(frame: CGRect(x: 22, y: 490, width: 75, height: 75))
+            myFotosImageView.image = myFotosImage
+            view.addSubview(myFotosImageView)
+
+            nameLabelAlexey = UILabel(frame: CGRect(x: 105, y: 527, width: 209, height: 20))
+            nameLabelAlexey.text = ""
+            if let customFont = UIFont(name: fontName, size: 14) {
+                nameLabelAlexey.font = customFont
+                view.addSubview(nameLabelAlexey)
+            }
+        }
+    }
+
+    // MARK: Private Methods
+
+    @objc private func moveForvard() {
+        let modalViewController = InformationScreen()
+        modalViewController.dataDelegate = self
+        modalViewController.modalPresentationStyle = .automatic
+        present(modalViewController, animated: true, completion: nil)
+    }
+
+    // MARK: Private Methods
+
+    private func setupLabelNameLastName() {
+        nameLabelVerona = UILabel(frame: CGRect(x: 102, y: 119, width: 209, height: 20))
+        nameLabelVerona.text = "Helena Link"
+        nameLabelVerona.shadowColor = .black
+        nameLabelVerona.shadowOffset = CGSize(width: 1, height: 1)
+        if let customFont = UIFont(name: fontName, size: 16) {
+            nameLabelVerona.font = customFont
+        }
+        view.addSubview(nameLabelVerona)
+
+        nameLabelVerona = UILabel(frame: CGRect(x: 103, y: 214, width: 209, height: 20))
+        nameLabelVerona.text = "Verona Tusk"
+        nameLabelVerona.shadowColor = .black
+        nameLabelVerona.shadowOffset = CGSize(width: 1, height: 1)
+        if let customFont = UIFont(name: fontName, size: 16) {
+            nameLabelVerona.font = customFont
+        }
+        view.addSubview(nameLabelVerona)
+
+        nameLabelAlex = UILabel(frame: CGRect(x: 103, y: 309, width: 209, height: 20))
+        nameLabelAlex.text = "Alex Smitch"
+        nameLabelAlex.shadowColor = .black
+        nameLabelAlex.shadowOffset = CGSize(width: 1, height: 1)
+        if let customFont = UIFont(name: fontName, size: 16) {
+            nameLabelVerona.font = customFont
+        }
+        view.addSubview(nameLabelAlex)
+
+        nameLabelTom = UILabel(frame: CGRect(x: 105, y: 404, width: 209, height: 20))
+        nameLabelTom.text = "Alex Smitch"
+        nameLabelTom.shadowColor = .black
+        nameLabelTom.shadowOffset = CGSize(width: 1, height: 1)
+        if let customFont = UIFont(name: fontName, size: 16) {
+            nameLabelVerona.font = customFont
+        }
+        view.addSubview(nameLabelTom)
+    }
+
+    // MARK: Private Methods
+
+    private func setuplLabel() {
```

Comment:
опечатка в слове setup
```suggestion
    private func setupLabel() {
```